src/annotations/items/text/TextCursor.h

```diff
@@ -52,7 +52,11 @@ Q_OBJECT
     int mPosition = 0;
     bool mIsVisible = false;
 
+    void moveCursorAtBeginning();
```

Comment:
hmm, should it be `moveCursorAtBeginning`or `moveCursorToBeginning`?

Reply:
Renamed.